src/apprt/gtk/Surface.zig

```diff
@@ -2298,3 +2343,39 @@ fn g_value_holds(value_: ?*c.GValue, g_type: c.GType) bool {
     }
     return false;
 }
+
+fn gtkPromptTitleResponse(source_object: ?*gobject.Object, result: *gio.AsyncResult, ud: ?*anyopaque) callconv(.C) void {
+    const dialog: *adw.AlertDialog = @ptrCast(source_object.?);
+    const self = userdataSelf(ud.?);
+
+    const response = dialog.chooseFinish(result);
+    if (std.mem.orderZ(u8, "ok", response) == .eq) {
+        const title_entry: *gtk.Entry = gobject.ext.cast(gtk.Entry, dialog.getExtraChild().?).?;
+        const title = std.mem.span(title_entry.getBuffer().getText());
+
+        // if the new title is empty and the user has set the title previously, restore the terminal provided title
+        if (title.len == 0) {
+            if (self.getTerminalTitle()) |terminal_title| {
+                self.setTitle(terminal_title, .user) catch |err| {
+                    log.err("Failed to set title: {}", .{err});
```

Comment:
This isn't really *formalized* anywhere but our log messages usually look like
```suggestion
                    log.err("failed to set title={}", .{err});
```

Reply:
Fixed all log messages.